cypress/integration/inventory/Item-status-date-updates.spec.js

```diff
@@ -0,0 +1,376 @@
+        Campuses.createViaApi(specialCampuse);
+        const specialLibrary = { ...Libraries.defaultUiLibraries.body };
+        Libraries.createViaApi(specialLibrary);
+        specialLibrary.campusId = specialCampuse.id;
+
+        effectiveLocation = { ...NewLocation.getDefaultUiLocation(Institutions.defaultUiInstitutions.body.id, specialCampuse.id, specialLibrary.id).body, servicePointIds: [effectiveLocationServicePoint.body.id], primaryServicePoint: effectiveLocationServicePoint.body.id };
+
+        userRequestPreferences = {
+          id: uuid(),
+          fulfillment: 'Delivery',
+          defaultDeliveryAddressTypeId: null,
+          defaultServicePointId: effectiveLocationServicePointId,
+          delivery: true,
+          holdShelf: true,
+          userId: userForDeliveryRequest.id,
+        };
+
+        ServicePoints.createViaApi(effectiveLocationServicePoint.body);
+        ServicePoints.createViaApi(notEffectiveLocationServicePoint.body);
+        cy.addServicePointToUser([effectiveLocationServicePoint.body.id, notEffectiveLocationServicePoint.body.id],
+          user.userId, effectiveLocationServicePoint.body.id);
+        Locations.createLocationViaApi(effectiveLocation)
+          .then(locations => {
+            orderLine.locations[0].locationId = locations.body.id;
+            itemLocation = locations.body.name;
+          });
+        cy.login(userProperties.username, userProperties.password)
+          .then(() => {
+            Orders.createOrderWithOrderLineViaApi(order, orderLine)
+              .then(orderNumber => {
+                createdOrderNumber = orderNumber;
+              });
+          });
+        cy.getUsers({ limit: 1, query: `"personal.lastName"="${user.username}" and "active"="true"` })
+          .then((users) => {
+            userBarcode = users[0].barcode;
+          });
+      });
+
+    cy.createTempUser([
+      permissions.checkoutAll.gui,
+      permissions.requestsAll.gui,
+    ])
+      .then(userProperties => {
+        userForDeliveryRequest = userProperties;
+        cy.getUsers({ limit: 1, query: `"username"="${userForDeliveryRequest.username}"` })
+          .then((users) => {
+            userForDeliveryRequest.barcode = users[0].barcode;
+            userRequestPreferences.userId = users[0].id;
+            cy.getAddressTypesApi({ limit: 1 }).then(addressTypes => {
+              userRequestPreferences.defaultDeliveryAddressTypeId = addressTypes[0].id;
+            })
+              .then(() => {
+                UpdateUser.updateUserAddress(users[0], [{ city: 'New York',
+                  addressTypeId: userRequestPreferences.defaultDeliveryAddressTypeId,
+                  countryId: 'US' }]);
+                cy.createUserRequestPreferencesApi(userRequestPreferences);
+              });
+          });
+      });
+  });
+
+  afterEach(() => {
+    cy.getInstance({ limit: 1, expandAll: true, query: `"items.barcode"=="${userItemBarcode}"` })
```

Comment:
think to remember related ids and send direct DELETE requests instead of work through additional get of instance